src/ui/reducers/library.js

```diff
@@ -1,10 +1,47 @@
-import app          from '../lib/app';
-import AppConstants from '../constants/AppConstants';
-import utils        from '../utils/utils';
+import * as app from '../lib/app';
+import types from '../constants/action-types';
+import utils from '../utils/utils';
+
+
+const initialState = {
+  tracks: {
+    library: {
+      all: null,
+      sub: null,
+    },
+    playlist: {
+      all: null,
+      sub: null,
+    },
+  },
+  tracksCursor: 'library', // 'library' or 'playlist'
```

Comment:
I'd move it to some enum so that we can refer to it as
```js
tracksCursor: cursorTypes.library
// or
tracksCursor: cursorTypes.playlist
```

Reply:
I agree, let's open an issue for that

Reply:
https://github.com/KeitIG/museeks/issues/372